### PR TITLE
[RHACS] Added module for updating custom certificate

### PR DESCRIPTION
**Pull request by gaurav-nelson**
- Added instructions for updating the custom certificate.

Preview at https://openshift-docs-git-update-central-cert-gnelson.vercel.app/openshift-acs/master/configuration/add-custom-certificates.html#update-custom-certificate-central_add-custom-cert

NOTES:

- This PR is for RHACS docs at https://docs.openshift.com/acs/welcome/index.html
- It doesn't require any special labels or milestones

### CODE REVIEW

**Comment by Github4Aaron** [comment]
looks good @gaurav-nelson

File: modules/update-custom-certificate-central.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[role="_abstract"]
+If you are using a custom certificate for Central, you can update it by following the instructions in this section.
+
+.Procedure
+. Delete the existing custom certificate's secret.
++
+[source,terminal]
+----
+$ oc delete secret central-default-tls-cert
```
Comment (ctauchen) [inline modules/update-custom-certificate-central.adoc:16]: Is `central-default-tls-cert` the standard name for all installations? Or is this an example, and users must supply their own secret name here?

If it is an example, I'd suggest making that explicit.

File: modules/update-custom-certificate-central.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+----
+. Create a new secret.
++
+[source,terminal]
+----
+$ oc -n stackrox create secret tls central-default-tls-cert \
```
Comment (ctauchen) [inline modules/update-custom-certificate-central.adoc:22]: Again, just checking:

Is central-default-tls-cert the standard name for all installations? Or is this an example, and users must supply their own secret name here?

If it is an example, I'd suggest making that explicit.

**Review by ctauchen — COMMENTED** [review]
This looks good, @gaurav-nelson. A few small questions below.

File: modules/update-custom-certificate-central.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/update-custom-certificate-central.adoc:16]: Thank you @ctauchen 

Those are the default secret names that RHACS generates.

File: modules/update-custom-certificate-central.adoc (hunk 2 above)
Comment (gaurav-nelson) [reply modules/update-custom-certificate-central.adoc:22]: PS: https://github.com/openshift/openshift-docs/pull/36361#discussion_r722061476

File: modules/update-custom-certificate-central.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: PROCEDURE
+[id="update-custom-certificate-central_{context}"]
+= Updating the custom certificate for an existing instance
+
+[role="_abstract"]
+If you are using a custom certificate for Central, you can update it by following the instructions in this section.
```
Comment (mikemckiernan) [inline modules/update-custom-certificate-central.adoc:9]: I've forgotten the term for unnecessary use of "to be", but I think you could go with "If you use a custom..."

For the second clause in the sentence, I think you can avoid self-referential text.  From ISG:

> Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". 

Maybe "..., you can update the certificate by performing the following procedure."

Rando free comment: Doesn't everyone know that you can skip the role assignment and the first para is automatically set as the abstract?

File: modules/update-custom-certificate-central.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+.Procedure
+. Delete the existing custom certificate's secret.
```
Comment (mikemckiernan) [inline modules/update-custom-certificate-central.adoc:12]: Through out the rest of the procedure, please add blank lines between steps.  I suspect this is more convention than technical.

Please end the instructional text with a colon.

**Review by mikemckiernan — COMMENTED** [review]
I was going to merge, but spotted two items to consider.  Up to you.

**Review by mikemckiernan — COMMENTED** [review]
/lgtm

**Pull request merged**